### DISCUSSION TITLE
[API] Export as "plain text" (Markdown or HTML)

### DISCUSSION THREAD

**Discussion by Mara-Li** [mProjectsCode/obsidian-meta-bind-plugin#213] [Ideas]
Hi!
I'm the developper of GitHub Publisher, and I would like to allow user to export their Metabind content into pure markdown when my plugin export files.

I searched in the code and the FAQ, and I can't found anything that help me to do that.

The idea is similar to the way I parse and convert dataview:
- Search for bloc of `meta-bind` and `INPUT`
- Convert the input into HTML / Markdown (depending of what is it, like slider or simple text)
- Replace the code block by the generated html
- Return the converted text

For the moment, I didn't find anything that is like an export as "plain text" (or HTML)

**Comment by mProjectsCode** [comment]
There is no functionality like that.
You can take a look at the [API](https://github.com/mProjectsCode/obsidian-meta-bind-plugin/blob/master/src/api/API.ts). Maybe you can use any of those functions.

**Reply by Mara-Li** [reply]
So maybe I can make a FR for that ? :D
